OKTA-804418 Deprecate Apps API: update /apps API links to Redocly (#5092)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2016/index.md

Once you have shared a credential between apps, you can list all the applications that are using
the same application key credential. <!-- OKTA-100925 -->

> For more information, see the [Apps API reference](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/listApplications).

Reviewer: ```suggestion
See the [Apps API reference](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/listApplications).
```

3. Share the credential (`kid`) with one or more target apps.
4. Use the new credential in the target app.

> For more detailed instructions, see ["Clone Key Credential for Application"](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationSSOCredentialKey/#tag/ApplicationSSOCredentialKey/operation/cloneApplicationKey)

Reviewer: ```suggestion
For more detailed instructions, see [Clone key credential for app](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationSSOCredentialKey/#tag/ApplicationSSOCredentialKey/operation/cloneApplicationKey)
```

> and ["Update Key Credential for Application"](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationSSOCredentialKey/#tag/ApplicationSSOCredentialKey/operation/generateApplicationKey).

Reviewer: ```suggestion
and [Update key credential for app](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationSSOCredentialKey/#tag/ApplicationSSOCredentialKey/operation/generateApplicationKey).
```

using the [**App Integration Wizard**](https://help.okta.com/okta_help.htm?id=ext_Apps_App_Integration_Wizard-saml)
in the administrator UI.

> For more information about creating custom apps with the API, see [Apps API: Add Custom SAML Application](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/createApplication).

Reviewer: ```suggestion
For more information about creating custom apps with the API, see [Create an app](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Application/#tag/Application/operation/createApplication).
```
